src/storage/heap_file.c

```diff
@@ -25231,8 +25306,9 @@ heap_get_visible_version_internal (THREAD_ENTRY * thread_p, HEAP_GET_CONTEXT * c
  * Note: It is expected to have the home page fixed and also the forward page in case of relocation.
  */
 static int
-heap_update_set_prev_version (THREAD_ENTRY * thread_p, const OID * oid, PGBUF_WATCHER * home_pg_watcher,
-			      PGBUF_WATCHER * fwd_pg_watcher, LOG_LSA * prev_version_lsa)
+heap_update_set_prev_version (THREAD_ENTRY * thread_p, const HFID * hfid, const OID * oid,
+			      PGBUF_WATCHER * home_pg_watcher, PGBUF_WATCHER * fwd_pg_watcher,
+			      LOG_LSA * prev_version_lsa)
```

Comment:
Nit:
```suggestion
			      const LOG_LSA * prev_version_lsa)
```
The signature of `or_mvcc_set_log_lsa_to_record` must also be changed to take a `const LOG_LSA *lsa`.
Up to you if you want to make the change or not.